internal/usage/usage.go

```diff
@@ -34,7 +34,7 @@ const (
 	NExamples                = "Maximum number of example queries to provide that will be improved by a suggested index."
 	NIndexes                 = "Maximum number of indexes to suggest."
 	MDBVersion               = "Major MongoDB version of the cluster."
-	AuthDB                   = "Authentication database name."
+	AuthDB                   = "Authentication database name. If the user authenticates with AWS IAM, x.509, or LDAP, this value should be $external. If the user authenticates with SCRAM-SHA, this value should be admin."
```

Comment:
this change affects ops manager/cloud manager docs as we are reusing the same docs and AWS IAM is not valid for those.
If you want to update this copy please create a new variable and use the new variable